File: Year 1/java progamming/Examples/Scope.java
class Scope
{
	final static String txt = "This is a global variable of the Scope class";

	public static void main ( String[] args )
	{
		String txt = "This is a local variable in the main method";
		System.out.println( txt );
		sub();

		System.out.println( Scope.txt );
	}

	public static void sub( )

	{	
		//String txt = "This is a local variable in the sub method";	
		System.out.println( txt );
	}
}

File: Year 3/Semester 1/project/Files/Examples/Snippet of Code/Php Code/php5code/scope.php
<!-- example for PHP 5.0.0 final release -->

<?php 
	$num;

	function make_triple($arg)
	{
	  global $num;
	  $num = $arg + $arg +$arg;
	  thrice();

	}

	function thrice()
	{
	  global $num;
	  echo("The value is $num");
	} 
?>

<html>
 <head>
  <title>Variable Scope</title>
 </head>
 <body>

  <h3> <?php make_triple(4); ?> </h3>

 </body>
</html>